.env.example

```diff
@@ -1,4 +1,4 @@
-OPENVPN_HOST=
+OVPN_HOST=
```

Comment:
to align with https://github.com/OpenVPN/terraform-provider-openvpn-cloud/blob/main/e2e/integration_test.go#L16